VEDA/settings.py

```diff
@@ -110,6 +110,15 @@
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = 'VEDA.wsgi.application'
 
+# See: http://getblimp.github.io/django-rest-framework-jwt/#additional-settings
+JWT_AUTH = {
+    # If the secret is wrong, JWT_VERIFY will raise a jwt.DecodeError telling you as such.
+    # You can still get at the payload by setting the JWT_VERIFY to False
+    'JWT_VERIFY': False,
+    'JWT_SECRET_KEY': SECRET_KEY,
```

Comment:
Please don't use django's `SECRET_KEY`.  When that key is used by multiple features, it makes it hard to re-key.  FYI @clintonb - in case he has any best practices to share since he's integrated other edX services using JWT.

Reply:
See https://github.com/edx/course-discovery/blob/master/course_discovery/settings/base.py#L356-L364.